tests/observability/metrics/conftest.py

```diff
@@ -580,6 +580,11 @@ def initial_metric_value(request, prometheus):
     return int(get_metrics_value(prometheus=prometheus, metrics_name=request.param))
 
 
+@pytest.fixture(scope="module")
```

Comment:
why you can't reuse existing initial_metric_value - you have one test only, so function scope should be ok

Reply:
making "module" scope fixtures when they not needed - does not make any sense, they keeps active for a long time (untill the pytest exists from that module)
If you have only one place where it is used - keep it function scope

regarding `vm_list` fixture - you can always check if it is used anywhere thru your IDE or by using grep
if you can't find any links - it is safe to change the scope